docs/intro/languages/other.mdx

```diff
@@ -1 +1,40 @@
-TODO
+Nitric is an open framework designed to support additional languages for multi-cloud development. Languages are supported through SDKs which provide Nitric APIs to cloud services and a FaaS micro-framework to build cloud functions.
+
+If you want add support for another language please contribute to the [GitHub Nitric](https://github.com/nitrictech) open source project. Please reach out to us, we would will be super happy to help.
```

Comment:
if you want **_to_** add support...
missing 'to'

Reply:
also 'we would will be...'